hnsendpoint.go

```diff
@@ -94,7 +117,7 @@ func modifyNetworkEndpoint(containerID string, endpointID string, request Reques
 	return nil
 }
 
-// GetHNSEndpointByID
+// Get the HNS Endpoint by ID

```

Comment:
Go style comments on exported functions should be something similar to "GetHNSEndpointByID gets an HNS endpoint by ID"